src/ol/format/Feature.js

```diff
@@ -3,7 +3,7 @@
  */
 import {assign} from '../obj.js';
 import {abstract} from '../util.js';
-import {get as getProjection, equivalent as equivalentProjection, transformExtent} from '../proj.js';
+import {get as getProjection, equivalent as equivalentProjection, transformExtent, get} from '../proj.js';
```

Comment:
Looks like we already import `get` here.

Reply:
Good catch. Fixed.